### PR TITLE
Add GDCR 2019 in Shanghai

### PR DESCRIPTION
**Pull request by dsdshcym**
- [x] URL needs to be filled

### CODE REVIEW

File: _data/events_gdcr2019/shanghai.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "title": "GDCR 2019 at Ekohe Shanghai",
+  "url": "",
```
Comment (dsdshcym) [inline _data/events_gdcr2019/shanghai.json]: I'm creating a page on meetup.com. I will update the url field in this PR after the page is created.

Comment (dsdshcym) [reply _data/events_gdcr2019/shanghai.json]: Hi @rachelcarmena!

I've added the url to the event page. Please check!

Comment (rachelcarmena) [reply _data/events_gdcr2019/shanghai.json]: Awesome, thanks!

**Comment by rachelcarmena** [comment]
Thanks @dsdshcym and welcome :tada:

**Pull request merged**